sense/loading.py

```diff
@@ -0,0 +1,174 @@
+            print(f'Weights found:\n{path_weights_string}')
+            weights = {}
+            for name, path in path_weights.items():
+                load_fn = load_backbone_weights if name == 'backbone' else load_weights
+                weights[name] = load_fn(path)
+
+            return model_config, weights
+        else:
+            print(f'Could not find at least one of the following files:\n{path_weights_string}')
+
+    raise Exception('ERROR - Weights file missing. To download, please go to '
+                    'https://20bn.com/licensing/sdk/evaluation and follow the '
+                    'instructions.')
+
+
+def load_weights(checkpoint_path: str):
+    """
+    Load weights from a checkpoint file.
+
+    :param checkpoint_path:
+        A string representing the absolute/relative path to the checkpoint file.
+    """
+    return torch.load(checkpoint_path, map_location='cpu')
+
+
+def load_weights_from_resources(checkpoint_path: str):
+    """
+    Load weights from a checkpoint file located in the resources folder.
+
+    :param checkpoint_path:
+        A string representing the absolute/relative path to the checkpoint file.
+    """
+    checkpoint_path = os.path.join(RESOURCES_DIR, checkpoint_path.split(f'resources{os.sep}')[-1])
+    try:
+        return load_weights(checkpoint_path)
+
+    except FileNotFoundError:
+        raise FileNotFoundError('Weights file missing: {}. '
+                                'To download, please go to '
+                                'https://20bn.com/licensing/sdk/evaluation and follow the '
+                                'instructions.'.format(checkpoint_path))
+
+
+def load_backbone_weights(checkpoint_path: str):
+    """
+    Load backbone weights from a checkpoint file, unless Travis is used. Raises an error pointing
+    to the SDK page in case weights are missing.
+
+    :param checkpoint_path:
+        A string representing the absolute/relative path to the checkpoint file.
+    """
+    if not using_travis():
+        return load_weights_from_resources(checkpoint_path)
+    else:
+        print('Weights are not loaded on Travis.')
+
+
+def build_backbone_network(selected_config: ModelConfig, weights: dict):
+    """
+    Creates a backbone network and load provided weights, unless Travis is used.
+
+    :param selected_config:
+        An instance of ModelConfig, specifying the backbone architecture name.
+    :param weights:
+        A model state dict.
+    :return:
+        A backbone network, with pre-trained weights.
+    """
+    backbone_network = getattr(backbone_networks, selected_config.model_name)()
+    if not using_travis():
+        backbone_network.load_state_dict(weights)
+    backbone_network.eval()
+    return backbone_network
+
+
+def using_travis():
+    """
+    Returns True if Travis is currently being used.
+    """
+    return os.getenv('TRAVIS', False) == 'true'
```

Comment:
How about making this a constant `TRAVIS_ENV` on top of the file? Or does it change at runtime?